tasks/columns.py

```diff
@@ -0,0 +1,60 @@
+import json
+from collections import OrderedDict
+
+
+CONDITIONS = "conditions"
+EXCEPTIONS = "exceptions"
+
+
+class ColumnsDeclarations:
```

Comment:
Probably a bit ambitious to put this in the main directory for now. I would move it inside `us.census` for now, as it is only used there at the moment. Once we have the requirements for this metamagic a bit more clear, we can work in a generic approach. Probably after merging this and UK.

Reply:
Of course it's ambitious! ;) Ok, I'm moving this inside `us.census` until we discuss the declarative approach